cmd/binfmt/main.go

```diff
@@ -148,7 +163,14 @@ func run() error {
 		defer syscall.Unmount(mount, 0)
 	}
 
-	for _, name := range parseArch(toUninstall) {
+	var uninstallArchs []string
+	if strings.HasSuffix(toUninstall, "-") {
```

Comment:
This seems like weird logic. Can we just make the parameter a pattern and users would do `--uninstall qemu-*`

Reply:
Yes a glob pattern should be better